### DISCUSSION TITLE
HIDP constants

### DISCUSSION THREAD

**Discussion by Slion** [microsoft/CsWin32#583] [Q&A]
I'm trying to use [`HidP_GetUsageValueArray`](https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/hidpi/nf-hidpi-hidp_getusagevaluearray). The returned value is declared as NTSTATUS, are the constant values it can take declared somewhere? If I put them in `NativeMethods.txt` is says it can't find them.

**Comment by AArnott** [comment]
The constants are missing from the metadata. I've filed an issue for that: https://github.com/microsoft/win32metadata/issues/964
